.github/workflows/ci.yml

```diff
@@ -96,6 +96,19 @@ jobs:
         RUST_BACKTRACE: 1
         CARGO_TERM_COLOR: always
         LIBGIT2_NO_VENDOR: ${{ matrix.LIBGIT2_NO_VENDOR || '0' }}
+    # Try to catch packaging issues that will prevent installation.
+    - name: Install
+      run: >-
```

Comment:
I'm assuming `>-` does the right thing, but I can't look up the documentation easily (thanks YAML).

Reply:
see https://stackoverflow.com/a/65808412 and https://yaml-multiline.info/